README.md

```diff
@@ -25,12 +25,80 @@ startrek_token: your_token_here
 python3 -m prpr.main
 ```
 
+Также удобно создать bash alias, 
+чтобы можно было запускать из любого местоположения, например, так:
+```
+alias prpr='cd /path/to/dir/prpr/ && source venv/bin/activate && python -m prpr.main'
+```
+
+### Опции запуска
+<details>
+  <summary>Развернуть список</summary>
+
+```
+usage: main.py 
+            [-h] 
+            [-m {default,all,open,closed}] 
+            [-p PROJECTS [PROJECTS ...]] 
+            [-n NO] 
+            [-s STUDENT] 
+            [-o] 
+            [-v]
+```
+
+#### optional arguments:
+
+* -h, --help            show this help message and exit
+* -o, --open            Open the first homework in the list in two tabs of default browser: tracker and reviewer 
+* -v, --verbose
+
+#### filters:
+
+  these allow to specify the subset of homeworks to be displayed, can be composed
+
+*  -m {default,all,open,closed}, --mode {default,all,open,closed} filter by status mode
+   * default: in review, open or on the side of user
+   * open: in review or open
+   * closed: resolved or closed 
+   * all: all, duh
+
+*  -p PROJECTS [PROJECTS ...], --projects PROJECTS [PROJECTS ...]
+                        the numbers of projects to be shown; multiple space-separated values are accepted
+
+*  -n NO, --no NO        the no of the homework to be shown, all other filters are ignored
+
+*  -s STUDENT, --student STUDENT
+                        the substring to be found in the student column, mail works best
+
+</details>
+
+### Примеры использования опций запуска
+<details>
+  <summary>Развернуть список</summary>
+   
+`python -m prpr.main -p 1 2` - вывести только 1 и 2 проекты
+
+`python -m prpr.main -n 100 -o` - открыть в браузере работу № 100 
```

Comment:
```suggestion
Открыть в браузере работу № 100:

```bash
python -m prpr.main --no 100 --open` 
```

Reply:
Please consider this format. Command-line snippets should always be put in fenced code blocks because GitHub augments them with copy icons. Really handy.